tests/src/python/CMakeLists.txt

```diff
@@ -143,6 +143,7 @@ ADD_PYTHON_TEST(PyQgsPointDisplacementRenderer test_qgspointdisplacementrenderer
 ADD_PYTHON_TEST(PyQgsPostgresDomain test_qgspostgresdomain.py)
 ADD_PYTHON_TEST(PyQgsProcessingRecentAlgorithmLog test_qgsprocessingrecentalgorithmslog.py)
 ADD_PYTHON_TEST(PyQgsProcessingInPlace test_qgsprocessinginplace.py)
+ADD_PYTHON_TEST(PyQgsProcessingCheckValidity test_qgsprocessing_checkvalidity.py)
```

Comment:
Shouldn't this belong alongside the processing tests in the processing subfolder?